node/src/actor_system_factory.rs

```diff
@@ -1605,6 +1616,39 @@ mod tests {
         system.run();
     }
 
+    #[should_panic(expected = "1: Automap failure: prefixProtocolError(\"Booga\")")]
```

Comment:
I don't understand why one couldn't insert a space in between the prefix and the actual error, but if you insist on this I believe you do based on an actuality which I'm not getting at the moment.

Reply:
I think it's because in at least one case there isn't any prefix--by which I mean the prefix is `""`-- and leaving out the space means that the error message won't begin with a space in that case.

Reply:
Yeah, I just hope the true prefixes ends with a space. Would look better definitely.